src/sentry/static/sentry/app/views/organizationIntegrations/integrationListView/test.tsx

```diff
@@ -0,0 +1,412 @@
+import compact from 'lodash/compact';
```

Comment:
@NisanthanNanthakumar I am not a fan of naming these files `test` and `control`. I think `IntegrationDirectory` would be a more logical name.

Reply:
@scefali Isn't the overall component called the Integration Directory. I thought we're just testing new views

Reply:
@NisanthanNanthakumar The original component is called `OrganizationIntegrations`. We never called it a directory before.